docs/layouts/shortcodes/region_ips.html

```diff
@@ -0,0 +1,81 @@
+<!-- INDIVIDUAL REGION TABLE SHORTCODE -->
+{{ $region := .Get "region" }}
+{{ $data := index $.Site.Data.regions $region }}
+<!-- Main region table -->
+<table class="region-table">
+    <tbody class="region-table-body">
+      <!-- Header -->
+      <tr class="region-table-header">
+        <th>Region</th>
+        <th>Outbound IPs</th>
+        <th>Inbound IPs</th>
+      </tr>
+      <!-- List individual regions -->
+      {{ range $data }}
+      <!-- Give the region a row -->
+      <tr>
+        <!-- Region overview -->
+        <td style="background: #f4f2f3;">
+            <!-- Name, region identifier, location guarantee -->
+            <table class="region-subtable">
+                <tbody class="region-subtable-body">
+                    <tr>
+                        {{ $regionAnchor := anchorize ( printf "%s %s" .name .label ) }}
+                        <td>
+                            <h4><a href="#{{ $regionAnchor }}" id="{{ $regionAnchor }}" aria-label="Anchor">{{ .name }}</a></h4>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><code style="background: white;">{{ .label }}</code></td>
+                    </tr>
+                    <!-- Data location guarantee when relevant -->
+                    {{ if .location_guarantee }}
+                    <tr>
+                        <td class="guarantee-note">(<a href="#glossary">Data location guarantee</a>)</td>
```

Comment:
```suggestion
                        <td class="guarantee-note">(<a href="#glossary">Data Location Guarantee</a>)</td>
```

To fit the glossary. Or change it there.

Reply:
related to your separate comment, we've resolved to remove the definition entirely. So alongside cleanup, which I'd be curious to see what you think of, this PR removes the definition rather than giving it Glossary space.